src/tests/office_supplies/TestInkToner.java

```diff
@@ -0,0 +1,50 @@
+package office_supplies;
+
+import common.Common;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.WebDriver;
+import pages.office_supplies.InkTonerPage;
+import pages.office_supplies.OfficeSupplies;
+
+/**
+ * Created by ddantas on 4/3/2017.
+ */
+public class TestInkToner {
+
+   protected WebDriver driver;
+
+
+
+    @Before
+    public void setUp() throws Exception {
+        Common cm = new Common();
+
+        driver = cm.getDriver();
+    }
+
+    @Test
+    public void verifyHeaderText() throws Exception {
+
+        InkTonerPage inkTonerPage = new InkTonerPage(driver);
+
+        inkTonerPage = inkTonerPage.navigateToInkTonerpage();
```

Comment:
line 32 instantiate inkTonerPage class and line 34 uses super class method to navigate to particular page. Yes, in  line 32 , I instantiating inkTonerPage class but does not allow me to use super class method. Therefore, I used line 34 to link super class method to child class.